viz/latent_traversals.py

```diff
@@ -55,7 +55,7 @@ def traverse_line(self, sample_latent_space=None, idx=None, size=5):
             # Sweep over linearly spaced coordinates transformed through the
             # inverse CDF (ppf) of a gaussian since the prior of the latent
             # space is gaussian
-            cdf_traversal = np.linspace(0.1, 0.9, size)
+            cdf_traversal = np.linspace(0.05, 0.95, size)
```

Comment:
should be attrinutes of the class

Reply:
Agreed - I will include this in my next PR